src/vm/threads.h

```diff
@@ -697,13 +697,15 @@ EXTERN_C void __stdcall OnHijackFPTripThread();  // hijacked JIT code is returni
 
 void CommonTripThread();
 
+#ifdef WIN64EXCEPTIONS
```

Comment:
Since this is used at one place only, I don't think it is worth introducing the macro.